workspace_test.go

```diff
@@ -29,9 +31,9 @@ func TestPreparePublicRepo(t *testing.T) {
 		t.Errorf("|%v|", err)
 	}
 	defer os.RemoveAll(f)
-
+	basename := fmt.Sprintf("%v", time.Now().UnixNano())
```

Comment:
duplicate of https://github.com/MediaMath/grim/pull/55/files#diff-9fe6a2c7832721bba3826ce78a216ebeR185

Should be a function.